### DISCUSSION TITLE
Specify tags on /applications creation API endpoint

### DISCUSSION THREAD

**Discussion by dokime7** [coollabsio/coolify#3801] [Feature Requests]
Hi,

It should be useful to be able to define tags when creating application via API.

Thanks 🙌

**Comment by lukacsaron** [comment]
Huge upvote on this! Not just for creation, but for update as well.

**Comment by ilmari-h** [comment]
Would love this!
Is there currently any way to add tags to applications via the API? Patch or otherwise? Couldnt find myself.

Would be especially useful with the `/deploy` endpoint which allows deployments by tag. Currently managing big number of containers, all automated via the API, and have to call `/deploy` on each individually
